server/manager/src/main/java/org/apache/accumulo/manager/FateServiceHandler.java

```diff
@@ -421,9 +421,17 @@ public void executeFateOperation(TInfo tinfo, TCredentials c, long opid, FateOpe
         if (!canMerge)
           throw new ThriftSecurityException(c.getPrincipal(), SecurityErrorCode.PERMISSION_DENIED);
 
-        Manager.log.debug("Creating merge op: {} {} {}", tableId, startRow, endRow);
-        goalMessage += "Merge table " + tableName + "(" + tableId + ") splits from " + startRow
-            + " to " + endRow;
+        String startRowStr = startRow.toString();
```

Comment:
Take a look at `StringUtils.defaultIfBlank()`. This would end up looking like:
```
String startRowStr = StringUtils.defaultIfBlank(startRow.toString(), "-inf");
```

Reply:
Cool thanks.